CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+## How to contribute to HedgeDoc react-client
+
+#### Do you have questions about the project?
+
+* Ask any question about HedgeDoc in our [Matrix Support Channel][matrix-support] or our [Discourse][discourse].
```

Comment:
```suggestion
```

Reply:
what's the problem here? I don't understand.

Reply:
If you're looking for the actual https-links they are in the last few lines of the file…

Reply:
You can delete this bullet point. The second one is better.